frontend/src/app/components/ExperimentPage.js

```diff
@@ -341,6 +340,23 @@ export default class ExperimentPage extends React.Component {
     this.didScroll = false;
   }
 
+  // this is set to 100, to accomodate Tracking Protection
+  // which has been sending telemetry pings via installs from dev
+  // TODO: figure out a non-hack way to toggle user counts when we have
+  // telemetry data coming in from prod
+  renderInstallationCount(installation_count, title) {
+    if (installation_count <= 100) {
+      return (
+        <span data-l10n-id="userCountContainerAlt" className="bold" data-l10n-args={JSON.stringify({ title })}>
```

Comment:
@lmorchard just noticed i left the data-l10n-arg in there. i can get rid of this